reana/cli.py

```diff
@@ -202,16 +294,28 @@ def is_component_dockerised(component):
     Useful to skip some docker-related commands for those components that are
     not concerned, such as building Docker images for `reana-cluster` that does
     not provide any.
+
+    :param component: standard component name
+    :type component: str
+
+    :return: True/False whether the component is dockerisable
+    :rtype: bool
     """
     if os.path.exists(get_srcdir(component) + os.sep + 'Dockerfile'):
         return True
     return False
 
 
 def shorten_component_name(component):
```

Comment:
Duplicated of [line 169](https://github.com/reanahub/reana/pull/69/files#diff-bf19991d576d0bbbd8ae5e5182952755R169) function?

Reply:
Hmm, I don't see any duplication regarding True/False part?!

Reply:
```console
$ git grep -n -A 16 'def shorten_component_name(component):'
reana/cli.py:169:def shorten_component_name(component):
reana/cli.py-170-    """Return canonical short version of the component name.
reana/cli.py-171-
reana/cli.py-172-    Example: reana-job-controller -> r-j-controller
reana/cli.py-173-
reana/cli.py-174-    :param component: standard component name
reana/cli.py-175-    :type component: str
reana/cli.py-176-
reana/cli.py-177-    :return: short component name
reana/cli.py-178-    :rtype: str
reana/cli.py-179-    """
reana/cli.py-180-    short_name = ''
reana/cli.py-181-    parts = component.split('-')
reana/cli.py-182-    for part in parts[:-1]:
reana/cli.py-183-        short_name += part[0] + '-'
reana/cli.py-184-    short_name += parts[-1]
reana/cli.py-185-    return short_name
--
reana/cli.py:309:def shorten_component_name(component):
reana/cli.py-310-    """Return canonical short version of the component name.
reana/cli.py-311-
reana/cli.py-312-    Example: reana-job-controller -> r-j-controller
reana/cli.py-313-
reana/cli.py-314-    :param component: standard component name
reana/cli.py-315-    :type component: str
reana/cli.py-316-
reana/cli.py-317-    :return: short component name
reana/cli.py-318-    :rtype: str
reana/cli.py-319-    """
reana/cli.py-320-    short_name = ''
reana/cli.py-321-    parts = component.split('-')
reana/cli.py-322-    for part in parts[:-1]:
reana/cli.py-323-        short_name += part[0] + '-'
reana/cli.py-324-    short_name += parts[-1]
reana/cli.py-325-    return short_name
```

Reply:
I see, thanks :smile: was looking for True/False...